writeengine/bulk/we_bulkload.cpp

```diff
@@ -314,6 +314,84 @@ int BulkLoad::loadJobInfo(const string& fullName, bool bUseTempJobFile, int argc
           rc, MSGLVL_ERROR);
       return rc;
     }
+
+    // MCOL-5021
+    execplan::CalpontSystemCatalog::OID tableAUXColOid;
+    std::string tblName;
+    std::string curTblName = curJob.jobTableList[i].tblName;
+    string::size_type startName = curTblName.rfind('.');
+
+    if (startName == std::string::npos)
```

Comment:
Could you please add comment what is happening in the next few lines?

Reply:
@drrtuy I don't think any of the CS code assumes the FQN could be `database.schema.table`. I am reusing the same code here as in `XMLJob::fillInXMLDataAsLoaded`.

Reply:
Agree.